homework1.cpp

```diff
@@ -0,0 +1,72 @@
+#include <cmath>
+#include <iostream>
+#include <limits>
+#include <stdio.h>
+using namespace std;
+
+int dec_func(int a, int b, int c) {
+
+  int dec = b * b - (4 * a * c);
+
+  printf("%d", dec);
+
+  if (dec >= 0) {
+    return dec;
+  } else {
+    printf("There is no solution because of negative descriminant");
+    return 0;
+  }
```

Comment:
можно написать так
```
  if (dec < 0) {
    printf("There is no solution because of negative descriminant");
    return 0;
   }
   return dec;
```

Reply:
+